docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Minio to the Google Cloud Storage Gateway mode
+type: Tutorials
+---
+
+By default, you install Kyma with the Asset Store in Minio stand-alone mode. This tutorial shows how to set Minio to the Google Cloud Storage (GCS) Gateway mode using an [override](/root/kyma/#tutorials-helm-overrides-for-kyma-installation).
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [gcloud](https://cloud.google.com/sdk/gcloud/)
+- [Google Cloud Platform (GCP)](https://cloud.google.com) project
+
+## Steps
+
+You can set Minio to the GCS Gateway mode during Kyma installation, or switch to the GCS Gateway mode after Kyma installation. In both cases you need to create a Google service account that has a private key and the **Storage Admin** role permissions, apply a ConfigMap with an override onto a cluster or Minikube, and trigger Kyma installation. Follow these steps:
```

Comment:
we need a red note that buckets created in Minio without using Bucket CR are not recreated/migrated while switching to Minio Gateway